src/AccordionSection/AccordionHeader.js

```diff
@@ -20,6 +25,7 @@ const AccordionHeader = ({
   onClick,
   panelId,
   text,
+  chevronRight,
```

Comment:
i think `iconRight` is a better name for this prop, because the fact that it's a chevron is an implementation detail of `ShotAccordionHeaderIcon`, and this component doesn't know anything about that